apps/web/pages/success.tsx

```diff
@@ -847,7 +848,9 @@ export async function getServerSideProps(context: GetServerSidePropsContext) {
   }
 
   const bookingInfo = await prisma.booking.findFirst({
-    where,
+    where: {
+      id: bookingId,
+    },
```

Comment:
@gitstart @baileypumfleet This is overwriting the where condition and will break the success page for team and dynamic events. I will have to revert this commit.